src/ui/Avatar/Avatar.tsx

```diff
@@ -23,6 +23,18 @@ const AvatarRoot = styled(AvatarPrimitive.Root, {
   '&:hover': {
     border: '1px solid $alert',
   },
+  variants: {
```

Comment:
It looks like these were defined in a different way in `main` already -- please fix the merge conflict & use those, creating new variants as needed